config/samples/ramendr_v1alpha1_metrodr_drpolicy.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: ramendr.openshift.io/v1alpha1
+kind: DRPolicy
+metadata:
+  name: metrodr-drpolicy-sample
+spec:
+  schedulingInterval: "1h" # hourly
+  replicationClassSelector:
```

Comment:
Can probably remove the regional dr specific fields.

Reply:
i removed `schedulingInterval`